test/parallel/test-http-get-pipeline-problem.js

```diff
@@ -35,7 +35,8 @@ const server = http.Server(function(req, res) {
 
 
 server.listen(0, function() {
-  for (var i = 0; i < total; i++) {
+  let i;
+  for (i = 0; i < total; i++) {
```

Comment:
Weird, I could swear `const` loop variables once worked, but I see it's failing now. Your `let` suggestion sounds good.